dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -60,6 +66,26 @@
 """
 
 
+is_of_supported_var = None
+def is_of_supported(device_id=0):
+    global is_of_supported_var
+    if is_of_supported_var is not None:
```

Comment:
This way, we would only occasionally test the `fn.optical_flow`, is that correct? I.e. only if we get lucky and get the machine with OF supported.

I think resolving this is rather out of scope of this PR though